packages/xstate-vue/test/typegenTypes.test.tsx

```diff
@@ -77,25 +77,31 @@ describe('useMachine', () => {
     defineComponent({
       setup: () => {
         // @ts-expect-error
-        useMachine(machine, {});
-        useMachine(machine, {
+        useMachine(machine.provide({}));
+        useMachine(
```

Comment:
Some of the adjusted test cases here might not make sense after the recent changes. I'll re-review this and clean up what is redundant here